### DISCUSSION TITLE
animalview HTML template broken for animals that are bonded

### DISCUSSION THREAD

**Discussion by sccpets** [sheltermanager/asm3#1380] [Q&A / Help] [answered]
The display breaks for this template only on any bonded animals. part of the description ends up displaying above the picture and ends with a stray "/>".  The description and all the other information does display correctly after the pictures. I've looked through the template and am completely stumped.

Example of broken bonded animal display: https://us09b.sheltermanager.com/service?account=sp1505&method=animal_view&animalid=3901&template=animalview

Example of correctly displaying animal: https://us09b.sheltermanager.com/service?account=sp1505&method=animal_view&animalid=3751&template=animalview

**Comment by Tetzor** [comment, marked as answer]
Hi,

The problem is something in the animal description breaking the HTML. If you go into the header section of the animal template and change ANIMALCOMMENTS on line 11 to DESCRIPTIONATTR that will fix it.

Kind regards,

Jon

**Reply by sccpets** [reply]
Thank you so much!
